mappings/net/minecraft/screen/ScreenHandler.mapping

```diff
@@ -1,4 +1,128 @@
+	COMMENT Slots can be backed by an inventory, allowing the changes to be reflected to the
+	COMMENT persistent storage (like block entities) on the server. Clients manipulate the
+	COMMENT slots by issuing a "slot click" packet. "Clicking" a slot includes actions like
+	COMMENT picking up crafting result, shift-clicking stacks, swapping stacks between the
+	COMMENT inventory and the hotbar, or dropping stacks.
+	COMMENT
+	COMMENT <p>Screen handlers also contain a list of {@linkplain Property properties}
+	COMMENT that are used for syncing integers (e.g. progress bars) from the server to the client.
+	COMMENT Properties can also be used to sync an integer from the client to the server, although
+	COMMENT it has to be manually performed. If a property relies on other objects, like
+	COMMENT a value from a block entity instance, then the property can delegate its operations
+	COMMENT using {@link PropertyDelegate}. The delegate is passed when creating the screen handler.
+	COMMENT On the server, access to the property's value is delegated to the delegate (which in
+	COMMENT turn delegates to another object like a block entity instance).
+	COMMENT On the client, access to the property's value still uses the synced value.
+	COMMENT
+	COMMENT <p>{@link ScreenHandlerContext} allows running code on the server side only. Screen
+	COMMENT handlers are designed to be used on both sides; any action modifying the world has
+	COMMENT to be wrapped in a call to the context. Like with the property delegate, a context
+	COMMENT with the world is passed to the screen handler on creation on the server. On the
+	COMMENT server, the context executes the function with the world and the position. On the
+	COMMENT client, the context does nothing.
+	COMMENT
+	COMMENT <h2 id="usage">How to use screen handlers</h2>
+	COMMENT <h3 id="creation">Creation</h3>
+	COMMENT <p>To create a new screen handler, subclass {@link ScreenHandler}, create and register
+	COMMENT a new {@linkplain ScreenHandlerType screen handler type}, and associate it with
+	COMMENT a handled screen.
+	COMMENT
+	COMMENT <p>A subclass should have two constructors. One is for the server, and should take
+	COMMENT the {@code syncId} and inventories, property delegates, or contexts that are used.
+	COMMENT The {@link #syncId} is shared between the two sides. It is used to verify that a player
+	COMMENT has a specific screen (handler) open so that they can move items, for example.
+	COMMENT The inventories are used to back a slot so that any changes to a slot is reflected
+	COMMENT on the backing inventory, and vice versa. Property delegates and contexts bridge
+	COMMENT between the screen handler and other parts of the world; see above for more description.
+	COMMENT
+	COMMENT <p>The constructor should {@linkplain #addSlot add slots}, {@link #addProperties
+	COMMENT add properties from the delegate}, and store the property delegate and screen handler
```

Comment:
```suggestion
	COMMENT add properties from delegates}, and store the property delegates and screen handler
```

Reply:
This takes a single `PropertyDelegate`, a list of properties that are delegated. So the word "delegate" is singular. (In the same way a list is named "list" not "lists")

Reply:
Right, but screen handlers can have multiple property delegates (or none) if they call that method multiple times with different delegates. The current wording implies that all screen handlers will have exactly one PD, which isn't true.